histomicstk/segmentation/label/trace_boundaries.pyx

```diff
@@ -0,0 +1,21 @@
+"""
+Cython source code: trace_boundaries
+"""
+import cython
+
+import numpy as np
+cimport numpy as np
+
+from libcpp.vector cimport vector
+
+cdef extern from "trace_boundaries_opt.h":
+  vector[vector[vector[int]]] trace_boundary(vector[vector[int]] label, int)
+
+
+def trace_boundaries(np.ndarray[int, ndim=2, mode="c"] imLabel not None, connectivity=4):
+
+    cdef vector[vector[vector[int]]] output
+
+    output = trace_boundary(imLabel, connectivity)
```

Comment:
Are you padding to trace boundaries of object at image edges correctly? If not, you can do a 1-pixel padding for `imLabel` all round right before calling this function.